src/python/WMCore/WMBS/MySQL/Jobs/LoadForErrorHandler.py

```diff
@@ -52,28 +49,72 @@ class LoadForErrorHandler(DBFormatter):
     runLumiSQL = """SELECT fileid, run, lumi, num_events FROM wmbs_file_runlumi_map
                      WHERE fileid = :fileid"""
 
-    def formatJobs(self, result):
+    def getFileNoLocation(self, jobBinds, fileList,
```

Comment:
For me it is a little confusing the it updates fileList in place just because the method name. 
Either it returns fileList event though it changed in place or changing the method name to updateFileListWithNoLocation might be a little clearer.

Reply:
I agree! I'll fix it